lib/pages/chat/chat.dart

```diff
@@ -900,7 +853,7 @@ class ChatController extends State<Chat>
     );
     if (highlight) {
       await scrollController.highlight(
-        index,
+        index + 1,
```

Comment:
Why `index + 1`. Pls explain it

Reply:
because we have n + 1 messages, but only n events, so need to add 1